unit/test_disk_cache_filesystem.cpp

```diff
@@ -243,6 +333,56 @@ TEST_CASE("Test on disk cache filesystem with requested chunk at last of file",
 	REQUIRE(GetFileCountUnder(TEST_ON_DISK_CACHE_DIRECTORY) == 3);
 }
 
+// Requested chunk involves the end of the file. and the following request will only read already cached content.
```

Comment:
Not sure what's the different with "Test on disk cache filesystem with requested chunk the first, middle and last chunk"?